### ISSUE TITLE
What is the process of video input? There is no preprocessing code in your code, but Hugging Face spaces supports video input.

### ISSUE THREAD

**Issue by sdjhshbswp** [X-PLUG/mPLUG-Owl#62]

**Comment by MAGAer13** [comment]
Hi, the model on huggingface space is the advanced version of mPLUG-Owl which natively support video with temporal related module as input without treating video as multiple frames. The video is tokenized into 65 tokens as image. We will release it very soon.

**Comment by sdjhshbswp** [comment]
> Hi, the model on huggingface space is the advanced version of mPLUG-Owl which natively support video with temporal related module as input without treating video as multiple frames. The video is tokenized into 65 tokens as image. We will release it very soon.

But the paper doesn't mention this temporal-related module. Will this module be detailed in the new version of the paper coming soon?

**Comment by MAGAer13** [comment]
Yes, specially we add the local temporal modeling module proposed mPLUG-2, and add a trajectory learnable queries in visual abstractor module. We will include this in the new version of the paper.

**Comment by feymanwang** [comment]
> 

really need it !

**Comment by MAGAer13** [comment]
> really need it !

We will release the video version in this week!

**Comment by MAGAer13** [comment]
We would not update the paper but we will include the specification of the model's design in the video branch. The code and weight's will be released

**Comment by feymanwang** [comment]
> > really need it !
> 
> We will release the video version in this week!

Hi, did you release the video version ? I didn't see the updates in this project

**Comment by MAGAer13** [comment]
> > > really need it !
> > 
> > 
> > We will release the video version in this week!
> 
> Hi, did you release the video version ? I didn't see the updates in this project

Sorry for that. The code and weight is under reviewing process, we will release it once the review is done.

**Comment by LinB203** [comment]
Hi, this is a wonderful work!
I wonder that when will you release the video version ?

**Comment by Aritra02091998** [comment]
What is the standard input format for video for mPLUG-Owl in finetuning ? Should it be same like the image input as given in the tutorial?
